### PR TITLE
Returning expression constructed with `SimpleExpr`

### PR DESCRIPTION
**Pull request by billy1624**
## PR Info

- Dependencies:
  - #333

- Dependents:
  - SeaQL/sea-orm#756

## Adds

- [x] Add `ReturningClause::Exprs`: expressing returning in `Vec<SimpleExpr>`

### CODE REVIEW

**Review by ikrivosheev — COMMENTED** [review]
@billy1624, thank you! Some comments

File: src/query/returning.rs [hunk 1]
```diff
@@ -45,4 +46,21 @@ impl Returning {
         let cols: Vec<_> = cols.into_iter().map(|c| c.into_column_ref()).collect();
         ReturningClause::Columns(cols)
     }
+
+    /// Constructs a new [`ReturningClause::Exprs`].
+    pub fn expr<T>(&self, expr: T) -> ReturningClause
```
[the remark this answers is not in the document]
Comment (billy1624) [reply src/query/returning.rs:135]: Add :)

**Comment by billy1624** [comment]
Hey @tyt2y3, please review and merge

**Review by ikrivosheev — APPROVED** [review]
@tyt2y3 LGTM!

**Review by tyt2y3 — APPROVED** [review]

**Pull request merged**